doc/TRAINING.rst

```diff
@@ -361,25 +362,104 @@ If feature caching is enabled, these augmentations will only be performed on the
+
+  * **p**: probability value between 0.0 (never) and 1.0 (always) if a given sample gets augmented by this method
+
+  * **n**: number of intervals to mask
+
+  * **size**: number of frequency bands to mask per interval
+
+Multi domain augmentations
+--------------------------
+
+**Time mask augmentation** ``--augment time_mask[p=<float>,n=<int-range>,size=<float-range>,domain=<domain>]``
+  Sets time-intervals within the augmented samples to zero (silence) at random positions.
+
+  * **p**: probability value between 0.0 (never) and 1.0 (always) if a given sample gets augmented by this method
+
+  * **n**: number of intervals to set to zero
+
+  * **size**: duration of intervals in ms
+
+  * **domain**: data representation to apply augmentation to - "signal", "features" or "spectrogram" (default)
+
+
+**Dropout augmentation** ``--augment dropout[p=<float>,rate=<float-range>,domain=<domain>]``
+  Zeros random data points of the targeted data representation.
+
+  * **p**: probability value between 0.0 (never) and 1.0 (always) if a given sample gets augmented by this method
+
+  * **rate**: dropout rate ranging from 0.0 for no dropout to 1.0 for 100% dropout
+
+  * **domain**: data representation to apply augmentation to - "signal", "features" or "spectrogram" (default)
+
+
+**Add augmentation** ``--augment add[p=<float>,stddev=<float-range>,domain=<domain>]``
+  Adds random values picked from a normal distribution (with a mean of 0.0) to all data points of the targeted data representation.
+
+  * **p**: probability value between 0.0 (never) and 1.0 (always) if a given sample gets augmented by this method
+
+  * **stddev**: standard deviation of the normal distribution to pick values from
+
+  * **domain**: data representation to apply augmentation to - "signal", "features" (default) or "spectrogram"
+
+
+**Multiply augmentation** ``--augment multiply[p=<float>,stddev=<float-range>,domain=<domain>]``
+  Multiplies all data points of the targeted data representation with random values picked from a normal distribution (with a mean of 1.0).
+
+  * **p**: probability value between 0.0 (never) and 1.0 (always) if a given sample gets augmented by this method
+
+  * **stddev**: standard deviation of the normal distribution to pick values from
+
+  * **domain**: data representation to apply augmentation to - "signal", "features" (default) or "spectrogram"
+
 
 Example training with all augmentations:
 
 .. code-block:: bash
 
         python -u DeepSpeech.py \
           --train_files "train.sdb" \
-          --augmentations_per_epoch 10 \
+          --feature_cache ./feature.cache \
+          --cache_for_epochs 10 \
+          --epochs 100 \
           --augment overlay[p=0.5,source=noise.sdb,layers=1,snr=50:20~10] \
-          --augment overlay[p=0.2,source=voices.sdb,layers=10:6,snr=50:20~10] \
```

Comment:
I think its a good idea to show that you can use the flag multiple times here, for noise and speech with different params.